commit ad96a1a52e3361f4fbc2759ca0fef0e74a954471
Author: Ravindu Deshan <droidcasts@protonmail.com>
Date:   Thu Mar 27 00:08:36 2025 +0530

    kernel/module.c: Ignore symbols crc check
    
    mmc: disable CRC check for better performance
    
    Signed-off-by: Ravindu Deshan <droidcasts@protonmail.com>

diff --git a/drivers/mmc/core/core.c b/drivers/mmc/core/core.c
index f1678fd2e..f9c4ccecc 100755
--- a/drivers/mmc/core/core.c
+++ b/drivers/mmc/core/core.c
@@ -62,7 +62,7 @@ static const unsigned freqs[] = { 400000, 300000, 200000, 100000 };
  * performance cost, and for other reasons may not always be desired.
  * So we allow it it to be disabled.
  */
-bool use_spi_crc = 1;
+bool use_spi_crc = 0;
 module_param(use_spi_crc, bool, 0);
 
 static int mmc_schedule_delayed_work(struct delayed_work *work,
diff --git a/kernel/module.c b/kernel/module.c
index 0df5812ae..f815fdc82 100755
--- a/kernel/module.c
+++ b/kernel/module.c
@@ -1345,9 +1345,9 @@ static int check_version(const struct load_info *info,
 	return 1;
 
 bad_version:
-	pr_warn("%s: disagrees about version of symbol %s\n",
+	pr_warn("%s: disagrees about version of symbol, but ignore...%s\n",
 	       info->name, symname);
-	return 0;
+	return 1;
 }
 
 static inline int check_modstruct_version(const struct load_info *info,
